src/app/tools/point_shapes.h

```diff
@@ -75,6 +79,54 @@ class BrushPointShape : public PointShape {
       m_secondaryColor = loop->getSecondaryColor();
     }
     m_lastGradientValue = -1;
+    m_symmetryMode = (loop->getSymmetry()) ? loop->getSymmetry()->mode() : gen::SymmetryMode::NONE;
+  }
+
+  void setSymmetricCommpressedImages(const Brush* brush) {
+    Image* image = brush->image();
+    m_compressedImages[0].reset(new CompressedImage(image,
+                                                    brush->maskBitmap(),
+                                                    false));
+    switch(m_symmetryMode) {
+      case gen::SymmetryMode::HORIZONTAL:
+      case gen::SymmetryMode::VERTICAL:
+      {
+        doc::algorithm::FlipType flip =
+          (m_symmetryMode == gen::SymmetryMode::HORIZONTAL)? doc::algorithm::FlipType::FlipHorizontal :
```

Comment:
Minor change in style:
```suggestion
          (m_symmetryMode == gen::SymmetryMode::HORIZONTAL)? doc::algorithm::FlipType::FlipHorizontal:
```